run.sh

```diff
@@ -260,9 +260,9 @@ run_julia() {
         julia -e 'using Pkg; Pkg.activate("Related"); Pkg.instantiate()' &&
         if [ $HYPER == 1 ]; then
 
-            capture "Julia" hyperfine -r $runs -w $warmup --show-output "julia --project=Related -e \"using Related; main()\""
+            capture "Julia" hyperfine -r $runs -w $warmup --show-output "julia --startup-file=no --project=Related -e \"using Related; main()\""
```

Comment:
this is not really needed but it helps with local development because often a Julia user would have startup-files